blocks/init/src/Blocks/components/menu/assets/main-menu-a11y.js

```diff
@@ -11,21 +11,19 @@ export class AccessibleMenu {
 	}
 
 	addClassName(el, cls) {
-		if (!el.className.match(`(?:^|\\s)${cls}(?!\\S)`)) {
-			el.className += ` ${cls}`;
-		}
+		el.classList.add(cls);
 	}
 
 	hasClassName(el, cls) {
-		return !!el.className.match(`(?:^|\\s)${cls}(?!\\S)`);
+		return el.classList.contains(cls);
 	}
 
 	delClassName(el, cls) {
-		el.className = el.className.replace(new RegExp(`(?:^|\\s)${cls}(?!\\S)`), '');
+		el.classList.remove(cls);
 	}
 
 	tabCloseMenu() {
-		const navActive = this.mainMenu.querySelectorAll(`.${this.IS_FOCUSED_CLASS}`);
+		const navActive = this.mainMenu.querySelector(`.${this.IS_FOCUSED_CLASS}`);
 
 		for (let i = 0; i < navActive.length; i++) {
```

Comment:
If you changed the above to `querySelector` now `navActive` should no longer be an array, please test it out.

Reply:
I will test it out on the real theme in a few sec 👍🏼